bentoml/deployment/serverless/__init__.py

```diff
@@ -0,0 +1,232 @@
+                config['functions'][api.name] = {
+                    'handler': api.name,
+                    'events': [{'http': 'path'}] 
+                }
+        elif self.platform == 'aws-lambda' or self.platform == 'aws-lambda-py2':
+            config['provider']['name'] = 'aws'
+            for api in apis:
+                config['functions'][api.name] = {
+                    'handler': 'handler.' + api.name,
+                    'events': [
+                        {
+                           'http': {
+                               "path": '/' + api.name,
+                               "method": 'post'
+                           }
+                        }
+                    ] 
+                }
+        else:
+            raise BentoMLException(
+                'check serverless does not support platform %s at the moment' % self.platform)
+        yaml = YAML()
+
+        with TempDirectory() as tempdir:
+            saved_path = os.path.join(tempdir, 'serverless.yml')
+            yaml.dump(config, Path(saved_path))
+            with subprocess.Popen(['serverless', 'info'], cwd=tempdir, stdout=PIPE,
+                                stderr=PIPE) as proc:
+                # We don't use the parse_response function here. Instead of raising error, we will just return false
+                content = proc.stdout.read().decode('utf-8')
+                response = content.strip().split('\n')
+                logger.debug('Serverless response: %s', '\n'.join(response))
+                error = [s for s in response if 'Serverless Error' in s]
+                if error:
+                    return False
+                else:
+                    if display_info:
+                        logger.info('BentoML: %s', '\n'.join(response))
+                    return True
+
+    def delete(self):
+        if not self.check_status(display_info=False):
+            raise BentoMLException('No active deployment for service %s' % self.bento_service.name)
+
+        if self.platform == 'google-python':
+            provider_name = 'google'
+        elif self.platform == 'aws-lambda' or self.platform == 'aws-lambda-py2':
+            provider_name = 'aws'
+        config = {
+            "service": self.bento_service.name,
+            "provider": {
+                "name": provider_name,
+                "region": self.region,
+                "stage": self.stage
+            }
+        }
+        yaml = YAML()
+        with TempDirectory() as tempdir:
+            saved_path = os.path.join(tempdir, 'serverless.yml')
+            yaml.dump(config, Path(saved_path))
+
+            with subprocess.Popen(['serverless', 'remove'], cwd=tempdir, stdout=PIPE, stderr=PIPE) as proc:
+                response = parse_serverless_response(proc.stdout.read().decode('utf-8'))
+                logger.debug('Serverless response: %s', '\n'.join(response))
+                if self.platform == 'google-python':
+                    pass
```

Comment:
`pass` ?

Reply:
I need setup google cloud function and test